server/journeys/referral/on-behalf-of/view.njk

```diff
@@ -0,0 +1,7 @@
+{% block content %}
+
+  <h1>Referrals (On Behalf Of) asdadasdwq is under construction...</h1>
+  <p>Please check back later when there is content to view.</p>
+  <b
```

Comment:
Incomplete tag here. It'll get fixed when the proper view markup is in though

Reply:
I'm not sure how that happened, apologies and thanks!